### ISSUE TITLE
Usage of the code

### ISSUE THREAD

**Issue by DanielTakeshi** [BerkeleyAutomation/HSR_CORE#1]
Hi @mdlaskey 

What is the proper way to use and run the code? For instance, say I want to run this:

https://github.com/BerkeleyAutomation/HSR_CORE/blob/master/examples/read_camera_data.py

The `self.cam` should be `cam` and you may have to fiddle with the `sys.path` but after that it seems OK. I did the following:

- Turn on the HSR
- Connect to the HSR via the HSR-mode from the tutorials, so that the `rostopic list` command now includes the HSR camera sensors
- Run `python examples/read_camera_data.py` (or `python read_camera_data.py` depending on the sys.path)

However, if I try and analyze the `c_img` and `d_img` stuff, it's None, indicating that the rostopics do not seem to be subscribed to correctly.

TL;DR code usage? Thanks!
